### ISSUE TITLE
Failed to check registry.npmjs.org for updates: "timeout of 10000ms exceeded" 

### ISSUE THREAD

**Issue by ZiloFire902** [homebridge/homebridge#3530]
Hi! 
I kept on getting the above error on my logs whenever I went to the plugins page of the Homebridge UI, and when I go to install a plugin, even on the terminal. There was this other post which suggested going to virtual machine (i'm running the server on windows) and doing as follows:               

> if you're still having this issue, it's not Homebridge, but OS config related.
> 
> My solution:
> 
> 1. SSH into PI
> 2. `sudo nano /etc/sysctl.d/local.conf` _- this is a non-existing file at this point_
> 3. add a new line: `net.ipv6.conf.all.disable_ipv6=1`
> 4. ctrl-x, y, enter _- this will create the new file and save the modifications you just made_
> 5. `sudo reboot`
> 
> this solved it for me.
> 
> **TL;DR:** I could ping the resolved IP address but not registry.npmjs.org. Disabling IPv6 solved timeout issue as name resolution seemed to be behind the problem.
> 
> _Originally posted by @dsylex in https://github.com/homebridge/homebridge-config-ui-x/issues/1146#issuecomment-1443637414_

However, the file will not save as the 'file or directory does not exist.'
How can i fix it?
